packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -362,7 +369,8 @@ async function launchDesktopApp(app: OfficeApp, manifest: ManifestInfo, document
   }
 
   // for Outlook, open Outlook.exe; for other Office apps, open the document
-  if (app == OfficeApp.Outlook) {
+  let path: string;
+  if (manifest.manifestType === ManifestType.JSON && app == OfficeApp.Outlook) {
```

Comment:
This should revert back to the original.

Reply:
changed